keycloak/keycloak-admin-ui
# Add/edit/delete LDAP mappers

## Motivation
https://issues.redhat.com/browse/APPDUX-900

## Brief Description
Add ability to Create, Update, and Delete LDAP mappers.

Notes:
- You can't create either of the role-based mappers yet.
- Several of the group fields are not updating properly (shared form with roles).
- No cypress tests yet.

These will be addressed in upcoming PRs.
 
## Verification Steps
1. Select an existing User federation > LDAP provider.
2. Go to the mappers tab.
3. Verify that you can delete an existing mapper.
4. Verify that you can add a new mapper (one of the 9 non-role-based mappers) and that the mapper is populated in the mapper list, and that the values you chose are saved.
5. Verify that you can edit an existing mapper and your changes are saved when you navigate away and back.

## Checklist:

## Additional Notes
